### DISCUSSION TITLE
Student Developer Pack

### DISCUSSION THREAD

**Discussion by Prabal864** [community/community#147063] [GitHub Education]
### Hi everyone,

I have submitted my request for SDP and approved also on 11 Dec, 2024
When will I be able to use the Student developer pack ?
![Screenshot 2024-12-14 104606](https://github.com/user-attachments/assets/64bcfaf3-c639-44a7-8933-e92528a4b3a8)

**Comment by davevad93** [comment]
Hi @Prabal864 ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @Prabal864 was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: 
#132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268, #140287, #145640, #145995

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
